### ISSUE TITLE
[experimental] Retire directory

### ISSUE THREAD

**Issue by ericwindmill** [flutter/samples#2520]
Experimental samples are not appropriate for this repository, because Flutter developers can't use them as a guide _today_, which is one of the requirements for this repository. See [contributing guide](https://github.com/flutter/samples/blob/main/CONTRIBUTING.md) for more information.

**Comment by domesticmouse** [comment]
Part of the reason for this directory was the needs of the Material team to have a place for code at `master`/`main` because that is where they live. Retirement of this directory would cut off that source of maintenance. I'd suggest that you should discuss this with the owners of the various experimental samples before shuttering this.

**Comment by ericwindmill** [comment]
> Part of the reason for this directory was the needs of the Material team to have a place for code at `master`/`main` because that is where they live. Retirement of this directory would cut off that source of maintenance. I'd suggest that you should discuss this with the owners of the various experimental samples before shuttering this.

I plan on talking to all stakeholders before I act on any of the 'retire' issues I created yesterday. I've created the tickets so there's a natural place to have the conversations about individual samples. I fully expect that some of them will be just closed without updates

**Comment by zanderso** [comment]
FYI @jonahwilliams the text/font puzzle game demo lives here https://github.com/flutter/samples/tree/main/experimental/varfont_shader_puzzle.

**Comment by jonahwilliams** [comment]
We're not currently using this sample for anything.

**Comment by ericwindmill** [comment]
FYI - These samples can be moved if they're valuable for a reason I don't see and they run on the stable channel today.

**Comment by Piinks** [comment]
The date_planner app was checked in to experimental to run against master while we work on iOS improvements, I think the iOS team was also planning to use it for benchmarking. cc @jmagman 

We were about to check in rolodex as well in experimental to run against master as we finish polishing Cupertino.

**Comment by jmagman** [comment]
> The date_planner app was checked in to experimental to run against master while we work on iOS improvements, I think the iOS team was also planning to use it for benchmarking. cc @jmagman

The iOS team isn't planning on using this app for benchmarking, afaik.  Right @LouiseHsu?

**Comment by LouiseHsu** [comment]
Yep I wasnt planning on using it.

**Comment by ericwindmill** [comment]
Good call, @Piinks, I didn't consider where I'd move date_planner. I need to consider the best way to handle large demo apps that are in active development, but my gut says that I'm just going to move it out experimental and add explanation to the README that it's incomplete or tested against the master branch or whatever qualifiers I need to add..

FWIW The problem I'm trying to solve by deleting `experimental` is that its often a dumping grounds where things are checked in and then forgotten. I'm discouraging any samples that can't pass CI against stable as a rule, but all rules have exceptions 😄

**Comment by Piinks** [comment]
> The problem I'm trying to solve by deleting experimental is that it's often a dumping ground where things are checked in and then forgotten.

You have my full support. 💯 

If we need to find another home for rolodex and date_planner, can do!

**Comment by HosseinYousefi** [comment]
The pedometer example works on the stable channel even though it's on experimental. It's also maintained by me.

**Comment by HosseinYousefi** [comment]
> Can we move the pedometer out of experimental into the main directory?

SGTM!
